src_c/_sdl2/video.pyx

```diff
@@ -706,6 +706,31 @@ cdef class Texture:
         self.draw_internal(csrcrect, cdstrect, angle, originptr,
                            flipX, flipY)
 
+    def update(self, surface, area=None):
+        # https://wiki.libsdl.org/SDL_UpdateTexture
+        # Should it accept a raw pixel data array too?
+        """ Update the texture with Surface.
+        This is a fairly slow function, intended for use with static textures that do not change often.
+
+        If the texture is intended to be updated often,
+        it is preferred to create the texture as streaming and use the locking functions.
+
+        While this function will work with streaming textures,
+        for optimization reasons you may not get the pixels back if you lock the texture afterward.
+
+        :param surface: source Surface.
+        """
+
+        if not pgSurface_Check(surface):
+            raise error("update source should be a Surface.")
+
+
+        cdef SDL_Rect rect = pgRect_AsRect(area)
+        cdef SDL_Surface *surf = pgSurface_AsSurface(surface)
+
+        res = SDL_UpdateTexture(self._tex, &rect, surf.pixels, surf.pitch)
```

Comment:
> If the texture is intended to be updated often, it is preferred to create the texture as streaming and use the locking functions referenced below. While this function will work with streaming textures, for optimization reasons you may not get the pixels back if you lock the texture afterward.

https://wiki.libsdl.org/SDL_UpdateTexture

Maybe it's better to use `SDL_LockTexture()` here? At least if it's a streaming texture.